todo/src/reducers.js

```diff
@@ -0,0 +1,62 @@
+const initialState = {
+    todos : [],
+}
+
+function rootReducer(state = initialState, action) {
+
+    switch (action.type) {
+        case "ADD_TODO": 
+            console.log("In ADD_TODO action of reducer ");
```

Comment:
Again, be cautious when using strings instead of constant variable in you action types